helm/values.yaml

```diff
@@ -137,6 +137,8 @@ agent:
   config: # optional config that can be provided to override the defaultConfig
   defaultConfig:
     {
+      "agent": {
```

Comment:
nit: can prob remove the agent section entirely

Reply:
removed and moved the logic to the helper function